core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -301,37 +318,44 @@ class AdminApiServiceSpec extends ApiServiceSpec {
 
     runAndWait(rawlsUserQuery.save(testUser))
 
-    Get(s"/admin/billing/list/${testUser.userEmail.value}") ~>
-      sealRoute(services.adminRoutes) ~>
-      check {
-        assertResult(StatusCodes.OK) {
-          status
-        }
-        assertResult(Set.empty) {
-          responseAs[Seq[RawlsBillingProjectName]].toSet
-        }
-      }
+    val getUri: Uri = s"/admin/billing/list/${testUser.userEmail.value}"
+    val putUri: Uri = s"/admin/billing/${project1.projectName.value}/user/${testUser.userEmail.value}"
 
-    createBillingProject(project1)
+    val sealedInstrumentedAdminRoutes: Route = sealRoute(instrumentRequest { services.adminRoutes })
```

Comment:
Yes.  I have been considering adding a master instrumented route to be shared by *all* tests.

```
val allRoutesInstrumented = sealRoute(instrumentRequest {
  services.adminRoutes ~ services.billingRoutes   // etc
}
```